tests/plugins/edition.test.ts

```diff
@@ -1032,4 +1034,109 @@ describe("edition", () => {
     model.dispatch("CYCLE_EDITION_REFERENCES");
     expect(model.getters.getCurrentContent()).toBe(`=${composerSheetName}!A1`);
   });
+
+  describe("Localized numbers and formulas", () => {
+    let model: Model;
+    beforeEach(() => {
+      model = new Model();
+    });
+
+    function editCell(model: Model, xc: string, content: string) {
+      selectCell(model, xc);
+      model.dispatch("START_EDITION", { text: content });
+      model.dispatch("STOP_EDITION");
+    }
+
+    describe("Number litterals", () => {
+      test("Decimal number detected with decimal separator of locale", () => {
+        editCell(model, "A1", "3,14");
+        expect(getEvaluatedCell(model, "A1").type).toBe(CellValueType.text);
+        expect(getEvaluatedCell(model, "A1").value).toBe("3,14");
+
+        updateLocale(model, { ...DEFAULT_LOCALE, decimalSeparator: "," });
+        editCell(model, "A2", "3,14");
+        expect(getEvaluatedCell(model, "A2").type).toBe(CellValueType.number);
+        expect(getEvaluatedCell(model, "A2").value).toBe(3.14);
+      });
+
+      test("Decimal numbers with dots are still detected in other locales", () => {
```

Comment:
you can add a comment saying this is not a functional requirement but rather a consequence/limitation of the implementation.